pageserver/src/tenant/throttle.rs

```diff
@@ -93,18 +94,34 @@ where
                 }
             })
             .collect();
+
+        // how frequently we drain a single token on average
+        let time_cost = refill_interval / refill_amount.get() as u32;
```

Comment:
This conversion to u32 is lossy, let's not do that. Will think of something.

Reply:
updated the config to use u32s